src/ORM/Rule/IsUnique.php

```diff
@@ -77,7 +77,7 @@ protected function _alias($alias, $conditions)
     {
         $aliased = [];
         foreach ($conditions as $key => $value) {
-            $aliased["$alias.$key"] = $value;
+            $aliased["$alias.$key IS"] = $value;
```

Comment:
@ndm2 Yes. We made the choice that using IS as an explicit way to have is null work was better than the implicit behaviour in 2.x